src/test/java/itest/GraphQLIT.java

```diff
@@ -460,8 +461,11 @@ void testDeleteMutation() throws Exception {
         query.put(
                 "query",
                 "query { targetNodes(filter: { annotations: \"PORT == 9093\" }) { recordings {"
-                    + " active { name doDelete { name } } archived { data { name doDelete { name }"
-                    + " } aggregate { count } } } } }");
+                        + " active { data { name doDelete { name }"
+                        + " } aggregate { count } }"
+                        + " archived { data { name doDelete { name }"
+                        + " } aggregate { count } }"
+                        + " } } }");
```

Comment:
Do the sample queries in https://github.com/cryostatio/cryostat/tree/main/docs/graphql also need updating similarly to this change?